### DISCUSSION TITLE
Условие публичности плейлиста

### DISCUSSION THREAD

**Discussion by Bondian** [Chimildic/goofy#167] [Помощь с алгоритмом] [answered]
Существует ли возможность прописывать зависимость публичности плейлиста от неких условий? Например, если количество треков в плейлисте меньше десяти, то плейлист - не публичный, и наоборот.

**Comment by Chimildic** [comment, marked as answer]
```js
let tracks = // ... 
Playlist.saveWithReplace({
  // ...
  public: tracks.length > 10,
});
```

**Reply by Bondian** [reply]
Спасибо!
